go.mod

```diff
@@ -5,6 +5,7 @@ go 1.14
 require (
 	contrib.go.opencensus.io/exporter/ocagent v0.7.0
 	github.com/DATA-DOG/go-sqlmock v1.5.0
+	github.com/Infoblox-CTO/ddi.common v2.1.2+incompatible
```

Comment:
The `atlas-app-toolkit` repo lives in our open source `infobloxopen` GitHub account. We can't cross-import with internal repos.